subworkflows/local/call_svs/tests/main.nf.test

```diff
@@ -141,7 +144,44 @@ nextflow_workflow {
 
     }
 
-    test("1 sample - [bam, bai], fasta, fai, [] - severus") {
+    test("1 sample - [bam, bai], fasta, fai, [], bed - sniffles") {
+
+        when {
+            workflow {
+                """
+                input[0] = Channel.of([
+                    [ id:'test', single_end:false, project: 'project' ], // meta map
+                    file(params.pipelines_testdata_base_path + 'testdata/HG002_PacBio_Revio.bam', checkIfExists: true),
+                    file(params.pipelines_testdata_base_path + 'testdata/HG002_PacBio_Revio.bam.bai', checkIfExists: true)
+                ])
+                input[1] = GUNZIP.out.gunzip
+                input[2] = SAMTOOLS_FAIDX.out.fai
+                input[3] = "sniffles"
+                input[4] = [[],[]]
+                input[5] = [
```

Comment:
I think this is okay (and I've done similarly), but I'm not sure the coordinates in this file will overlap with any variants we call in this test (?). 

If you wanted to, you could check the variants output in this test without any bed file, and then choose an appropriate region overlapping a variant. 

You can "create" a custom bed file like this:
https://github.com/nf-core/modules/blob/d7462e71f9129083ce10c3fe953ed401781e0ebd/modules/nf-core/modkit/pileup/tests/main.nf.test#L110-L112